commands/fly.go

```diff
@@ -26,7 +26,7 @@ type FlyCommand struct {
 	Watch   WatchCommand   `command:"watch"   alias:"w" description:"Stream a build's output"`
 
 	Containers ContainersCommand `command:"containers" alias:"cs" description:"Print the active containers"`
-	Hijack     HijackCommand     `command:"hijack"     alias:"intercept" alias:"i" description:"Execute a command in a container"`
+	Intercept  InterceptCommand  `command:"intercept"     alias:"intercept" alias:"i" description:"Execute a command in a container"`
```

Comment:
This makes the default command "intercept" but still leaves an alias of "hijack" for backwards compatibility.

Otherwise we cannot accept this until we do another major bump of concourse because of the breaking changes.